java/ql/lib/semmle/code/java/security/SensitiveResultReceiverQuery.qll

```diff
@@ -50,12 +51,45 @@ private class SensitiveResultReceiverConf extends TaintTracking::Configuration {
   }
 }
 
-/** Holds if there is a path from sensitive data at `src` to a result receiver at `sink`, and the receiver was obtained from an untrusted source `recSrc`. */
-predicate sensitiveResultReceiver(
+module SensitiveResultReceiverConfig implements DataFlow::ConfigSig {
+  predicate isSource(DataFlow::Node node) { node.asExpr() instanceof SensitiveExpr }
+
+  predicate isSink(DataFlow::Node node) {
+    exists(ResultReceiverSendCall call |
+      untrustedResultReceiverSend(_, call) and
+      node.asExpr() = call.getSentData()
+    )
+  }
+
+  predicate allowImplicitRead(DataFlow::Node n, DataFlow::ContentSet c) { isSink(n) and exists(c) }
+}
+
+module SensitiveResultReceiverFlow = TaintTracking::Global<SensitiveResultReceiverConfig>;
```

Comment:
Same as above: either make it private (as the original configuration was), or add QLDoc.